docs/DevelopmentPlan/DevelopmentPlan.tex

```diff
@@ -2,6 +2,7 @@
 
 \usepackage{booktabs}
 \usepackage{tabularx}
+\usepackage{parskip}
```

Comment:
Adding this package removes the indents on paragraphs and adds a space between paragraphs. I mainly wanted the space. If we want the indent we can add it back.

Reply:
I could move it into the common if everyone agrees to keep this.

Reply:
@TimChoy, @joyxiao99 thoughts? Would probably be nice to include in common so all docs have consistent indentation